<launch>
	<param name="iterations" value="1"/>
	<param name="template_filename" value="1_not_ordered_0.1mm.ply" />
	
	<node pkg="measure_node" type="measure_node" name="measure_node" output="screen"/>

    <include file="$(find gocator_3200)/launch/gocator_3200.launch"/> 

<!--     <node name="rviz" pkg="rviz" type="rviz" args="-d $(find gocator_publisher)/param/rviz/default.rviz" /> -->
</launch>